src/components/map/map.jsx

```diff
@@ -0,0 +1,17 @@
+import style from './map.module.scss';
+
+const Map = () => {
+  return (
+    <div className={style.infoPanel}>
```

Comment:
It looks confusing. I expected to see a map here, but all I see it's a info panel.